src/internal/sbom/catalog.go

```diff
@@ -24,6 +25,12 @@ var viewerAssets embed.FS
 var tranformRegex = regexp.MustCompile(`(?m)[^a-zA-Z0-9\.\-]`)
 
 func CatalogImages(tagToImage map[name.Tag]v1.Image, sbomDir, tarPath string) {
+	// Ignore SBOM creation if there the flag is set
+	if config.SkipSBOM {
```

Comment:
I personally prefer the pattern of checking this before the function is called. It's all personal preference though so I'm just talking to talk right now.

Reply:
I did that at first but it was an extra line of code and I'm evil 😭.  I don't mind if you want to change it though.